src/SlashCommands.tsx

```diff
@@ -472,13 +475,16 @@ export const Commands = [
                 );
             }
 
-            const topicEvents = room.currentState.getStateEvents('m.room.topic', '');
-            const topic = topicEvents && topicEvents.getContent().topic;
-            const topicHtml = topic ? linkifyAndSanitizeHtml(topic) : _t('This room has no topic.');
+            const content: MRoomTopicEventContent = room.currentState.getStateEvents('m.room.topic', '')?.getContent();
+            const topic = !!content ? ContentHelpers.parseTopicContent(content)
+                : { text: _t('This room has no topic.') };
+
+            const ref = e => e && linkifyElement(e);
+            const body = topicToHtml(topic.text, topic.html, ref);
```

Comment:
we want to support headings and such here though? (I suppose as an MVP it doesn't matter)

Reply:
Oh, I mean, yes!! I didn't realize you only wanted the stricter sanitation in the room header. 🤦‍♂️

I added another parameter to control the level of sanitizing and used it to allow the same tags allowed in messages here.